[0.434s] Invoking command in '/home/daisuke/ros2_ws/build/person_msgs': /usr/bin/cmake /home/daisuke/ros2_ws/src/person_msgs -DCMAKE_INSTALL_PREFIX=/home/daisuke/ros2_ws/install/person_msgs
[0.449s] -- Found ament_cmake: 0.9.12 (/opt/ros/foxy/share/ament_cmake/cmake)
[0.472s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
[0.472s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.618s] -- Found rosidl_default_generators: 1.0.1 (/opt/ros/foxy/share/rosidl_default_generators/cmake)
[0.620s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.623s] -- Found rosidl_adapter: 1.3.1 (/opt/ros/foxy/share/rosidl_adapter/cmake)
[6.465s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[6.518s] -- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
[6.675s] CMake Error at /opt/ros/foxy/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[6.675s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[6.675s]   person_msgs --arguments-file
[6.675s]   /home/daisuke/ros2_ws/build/person_msgs/rosidl_adapter__arguments__person_msgs.json
[6.675s]   --output-dir
[6.676s]   /home/daisuke/ros2_ws/build/person_msgs/rosidl_adapter/person_msgs
[6.676s]   --output-file
[6.676s]   /home/daisuke/ros2_ws/build/person_msgs/rosidl_adapter/person_msgs.idls)
[6.676s]   returned error code 1:
[6.676s] 
[6.676s]   Error processing 'unit8 age' of 'person_msgs/Person': ''unit8' is an
[6.676s]   invalid message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[6.676s] 
[6.676s]   Traceback (most recent call last):
[6.676s] 
[6.676s]     File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
[6.676s]       return _run_code(code, main_globals, None,
[6.677s]     File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
[6.677s]       exec(code, run_globals)
[6.677s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[6.677s]       sys.exit(main())
[6.677s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
[6.677s]       abs_idl_file = convert_to_idl(
[6.677s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[6.677s]       return convert_msg_to_idl(
[6.677s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[6.677s]       msg = parse_message_string(package_name, input_file.stem, content)
[6.677s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 516, in parse_message_string
[6.677s]       Type(type_string, context_package_name=pkg_name),
[6.677s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 276, in __init__
[6.677s]       super(Type, self).__init__(
[6.678s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 204, in __init__
[6.678s]       raise InvalidResourceName(
[6.678s] 
[6.678s]   rosidl_adapter.parser.InvalidResourceName: 'unit8' is an invalid message
[6.678s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[6.678s] 
[6.678s] Call Stack (most recent call first):
[6.678s]   /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[6.678s]   CMakeLists.txt:21 (rosidl_generate_interfaces)
[6.678s] 
[6.678s] 
[6.679s] -- Configuring incomplete, errors occurred!
[6.679s] See also "/home/daisuke/ros2_ws/build/person_msgs/CMakeFiles/CMakeOutput.log".
[6.682s] Invoked command in '/home/daisuke/ros2_ws/build/person_msgs' returned '1': /usr/bin/cmake /home/daisuke/ros2_ws/src/person_msgs -DCMAKE_INSTALL_PREFIX=/home/daisuke/ros2_ws/install/person_msgs
